tsconfig.json

```diff
@@ -19,6 +19,7 @@
     "**/es",
     "**/dist",
     "tests/e2e",
+    "**/__tests__/e2e",
```

Comment:
why do they fail TS?

Reply:
ah, they have their own tsconfig? I believe something is possible with nested configs, but I'm not fully sure. Could you put an ambient file at that module root and see if that declares the types as expected?

Reply:
Done ! Added XML reference tags in a `.d.ts` file